### ISSUE TITLE
Daemon support

### ISSUE THREAD

**Issue by eginhard** [radian-software/radian#509]
I usually use Emacs by starting a daemon that I leave running and then connect to it in graphical mode with `emacsclient -c`. This doesn't seem to work well with Radian.

When starting the daemon, `display-graphic-p` is `nil`, so the toolbar and other UI elements are not [disabled](https://github.com/radian-software/radian/blob/b86946f9d87060d81dc04c1d00a14c8b9d576739/emacs/radian.el#L5206) and will show in the clients. The zerodark theme also uses fallback colours because it sees that the daemon doesn't have true colour support although this wouldn't be an issue in the clients.

Is there a way to better support this workflow in Radian?

**Comment by raxod502** [comment]
Yeah, this is a known problem. I too have tried to use the daemon in the past but have abandoned it because of edge cases that seemed to be difficult to work around. I believe the only way to get things even halfway workable is to start the daemon as a graphical instance of Emacs. If you start it from the terminal, then it's locked to a broken tty compatibility mode. I have tried to figure out how to get both graphical and terminal clients to coexist in the same Emacs daemon, but to no avail.

**Comment by hrehfeld** [comment]
I've been using nothing but daemon for a few years.

I have problems with desktop restoring, and fonts only show after creating a second frame, but otherwise everything seems to work fine.

**Comment by raxod502** [comment]
@hrehfeld Did you have to do any particular configuration to make things work? Color themes often query `display-graphic-p` so I'm surprised this would work out of the box.
